packages/app-content-pages/src/screens/Resources/Resources.js

```diff
@@ -0,0 +1,271 @@
+import { Anchor, Box, Image, Paragraph, Text } from 'grommet'
+import { Trans, useTranslation } from 'next-i18next'
+import styled, { css } from 'styled-components'
+import SpacedHeading from '@zooniverse/react-components/SpacedHeading'
+import ZooniverseLogo from '@zooniverse/react-components/ZooniverseLogo'
+
+import PageLayout from '../../shared/components/PageLayout/layout.js'
+import Head from '../../shared/components/Head/index.js'
+import MaxWidthContent from '../../shared/components/MaxWidthContent/MaxWidthContent.js'
+import {
+  MobileHeading,
+  StyledHeading
+} from '../../shared/components/SharedStyledComponents/SharedStyledComponents.js'
+
+const StyledList = styled.ul`
+  margin-block-start: 0;
+  margin-block-end: 0;
+  padding-inline-start: 0;
+  list-style-type: none;
+  margin-bottom: 30px;
+`
+
+const ResourceLogo = styled(Box)`
+  height: 50px;
+  min-width: 50px;
+  max-width: 50px;
+  border-radius: 5px;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+  ${props =>
+    props.theme.dark
+      ? css`
+          border: solid 1px ${props.theme.global.colors['dark-2']};
+          box-shadow: 1px 1px 4px ${props.theme.global.colors['dark-2']};
+        `
+      : css`
+          border: solid 1px ${props.theme.global.colors['light-5']};
+          box-shadow: 1px 1px 4px ${props.theme.global.colors['light-4']};
+        `}
+`
+
+function ResourcesPage() {
+  const { t } = useTranslation('components')
+
+  return (
+    <>
+      <Head
+        description={t('Resources.description')}
+        title={t('Resources.title')}
+      />
+      <PageLayout>
+        <MobileHeading level='1' size='1.5rem'>
+          {t('Resources.title')}
+        </MobileHeading>
+        <Box pad={{ horizontal: 'medium' }} align='center'>
+          <MaxWidthContent>
+            <StyledHeading
+              color={{ light: 'brand', dark: 'accent-1' }}
+              level='1'
+              size='small'
+            >
+              {t('Resources.title')}
+            </StyledHeading>
+            <SpacedHeading
+              align='center'
+              size='1.5rem'
+              margin={{ top: 'medium' }}
+              textAlign='center'
+            >
+              {t('Resources.cite.heading')}
+            </SpacedHeading>
+
+            {/** FOR RESEARCHERS */}
+            <Text
+              size='1.13rem'
+              weight='bold'
+              color={{ light: 'black', dark: 'white' }}
+            >
+              {t('Resources.cite.subheadings.one')}
+            </Text>
+            <Paragraph margin={{ top: 'small', bottom: '0' }}>
+              <Trans
+                i18nKey='Resources.cite.paragraphs.one'
+                t={t}
+                components={[
+                  <Anchor
```

Comment:
Note that I'm not using `<Link/>` from 'next/link' on this page because of future plans to make the content page a react library rather than independent app